app/src/main/java/org/mozilla/fenix/components/metrics/MozillaProductDetector.kt

```diff
@@ -22,7 +22,7 @@ object MozillaProductDetector {
         FOCUS("org.mozilla.focus"),
 
         // Other products
-        LOCKBOX("org.mozilla.lockbox")
+        LOCKWISE("org.mozilla.lockwise")
```

Comment:
The application id hasn't changed!
https://play.google.com/store/apps/details?id=mozilla.lockbox

Reply:
Thanks for the heads up. Weird that it's failing to find it installed, then 😕

Reply:
Note that it's `mozilla.lockbox` without `org` :)

Reply:
Yes, just saw that. Thanks @pocmo